src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$color-blue-accent: rgba(0, 172, 220, 1);
+$border-color: #f3f3f3;
+$txt-color: #060b35;
+$color-main-accent: #060b35;
+$background-color: #fff;
```

Comment:
Think this should be just `$color-white` variable because it just point to white color